src/components/media/media-list.jsx

```diff
@@ -117,13 +124,59 @@ function MediaList({ fromModal, handleSelected }) {
     if (mediaToDelete.length > 0) {
       setIsDeleting(true);
       const toDelete = mediaToDelete.splice(0, 1).shift();
+      setLoadingMessage(t("media-list.loading-messages.deleting-media"));
       const toDeleteId = idFromUrl(toDelete["@id"]);
       DeleteV1Media({ id: toDeleteId });
     } else if (isDeleteSuccess) {
+      // If delete is a success, the list is reloaded, and a success message is saved in local storage for later use.
+      localStorage.setItem(
+        "messages",
+        JSON.stringify([
+          ...localStorageMessages,
+          t("media-list.success-messages.media-delete"),
+        ])
+      );
+      // @TODO: refetch
       window.location.reload(false);
     }
   }, [mediaToDelete, isDeleteSuccess]);
 
+  // Display success messages from successfully deleted slides.
+  useEffect(() => {
+    const messages = JSON.parse(localStorage.getItem("messages"));
```

Comment:
Why do we need to save to local storage?
is it because of the window reload?
We should handle this in a better way.

For now at least add a "TODO: Refactor this when Redux Toolkit cache refresh is set up."